app/controllers/hosts_controller.rb

```diff
@@ -912,7 +912,8 @@ def host_attributes_for_templates(host)
   end
 
   def csv_columns
-    [:name, :operatingsystem, :compute_resource_or_model, :hostgroup, :last_report]
+    Pagelets::Manager.pagelets_at("#{controller_name}/_list", 'hosts_table_column_content', filter: { selected: @selected_columns })
```

Comment:
nitpick: You are in `hosts_controller`, why do we need to generalize with `#{controller_name}`